src/context/ArchiveContext/ArchiveContext.jsx

```diff
@@ -0,0 +1,71 @@
+import { useContext, createContext, useState, useEffect } from 'react';
+import { useAxios } from '../../CustomHooks';
+
+const ArchiveContext = createContext();
+
+const ArchiveProvider = ({ children }) => {
+	const [archiveNotes, setArchiveNotes] = useState([]);
+	const { response: archiveResponse, operation: fetchArchive } = useAxios();
+
+	useEffect(() => {
+		if (archiveResponse != null) {
+			if (archiveResponse.archives) {
+				setArchiveNotes(archiveResponse.archives);
+			}
+		} else {
+			getArchive();
+		}
+	}, [archiveResponse]);
+	const getArchive = () => {
+		fetchArchive({
+			method: 'GET',
+			url: '/api/archives',
+			headers: {
+				authorization:
+					'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI0OGJmMzFiYi1iYWQyLTQ3N2EtYmRhNy1kZDc4NDIxNDZhODgiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.kH6xXhMMvQ0sgsCz7e8vrSh1pVddplgC3IMzUSlTFsQ',
+			},
+		});
+	};
+	const notesToArchive = (noteId, notes) => {
+		fetchArchive({
+			method: 'POST',
+			url: `/api/notes/archives/${noteId}`,
+			data: {
+				notes,
+			},
+			headers: {
+				authorization: localStorage.getItem('token'),
+			},
+		});
+	};
+
+	const deleteArchiveNotes = (noteId) => {
+		fetchArchive({
+			method: 'DELETE',
+			url: `/api/archives/delete/${noteId}`,
+			headers: {
+				authorization: localStorage.getItem('token'),
+			},
+		});
+	};
+
+	const archiveToNotes = (notesId) => {
```

Comment:
Suggestion: Better naming of function can be done here.
You have 2 functions `notesToArchive` and `archiveToNotes`.
Seeing the API call, I'm guessing the 2nd function undoes the action performed by the first function.
It would make more sense if `archiveToNotes` is named `UndoArchive`.

Reply:
addressed in vc